### PR TITLE
Store customerId in Stripe

### PR DESCRIPTION
**Pull request by mpeterss**
This remove the storage of the paymentId in Firebase. Instead we
will get the Id from Stripe. Each time we create a new user in
Stripe we store the customer email with Stripe. So this is used
to fetch the paymentId once needed.

### CODE REVIEW

File: payment-processor/src/main/kotlin/org/ostelco/prime/paymentprocessor/StripePaymentProcessor.kt [hunk 1]
```diff
@@ -59,6 +62,20 @@ class StripePaymentProcessor : PaymentProcessor {
...
 
+    override fun getPaymentProfile(userEmail: String): Either<PaymentError, ProfileInfo> {
+            val customerParams = mapOf(
+                    "limit" to "1",
+                    "email" to userEmail)
+            val customerList = Customer.list(customerParams)
+            if (customerList.data.isEmpty()) {
+                 return Either.left(NotFoundError("Could not find a payment profile for user  $userEmail"))
+            } else if (customerList.data.size > 1){
+                 return Either.left(NotFoundError("Multiple profiles for user $userEmail found"))
```
Comment (prasanthu) [inline payment-processor/src/main/kotlin/org/ostelco/prime/paymentprocessor/StripePaymentProcessor.kt]: Is this allowed in Stripe (having multiple profiles with same email ) ?

File: acceptance-tests/src/main/kotlin/org/ostelco/at/jersey/Tests.kt [hunk 2]
```diff
@@ -1,7 +1,6 @@
 package org.ostelco.at.jersey
 
 import org.junit.Test
-import org.ostelco.at.common.Firebase
```
Comment (prasanthu) [inline acceptance-tests/src/main/kotlin/org/ostelco/at/jersey/Tests.kt]: Did you remove the Firebase dependency from gradle settings

File: payment-processor/src/main/kotlin/org/ostelco/prime/paymentprocessor/StripePaymentProcessor.kt (hunk 1 above)
Comment (mpeterss) [reply payment-processor/src/main/kotlin/org/ostelco/prime/paymentprocessor/StripePaymentProcessor.kt]: I am not sure. The Documentation does not mention if it must be unique.
https://stripe.com/docs/api#create_customer
So I added a check incase :)

File: acceptance-tests/src/main/kotlin/org/ostelco/at/jersey/Tests.kt (hunk 2 above)
Comment (mpeterss) [reply acceptance-tests/src/main/kotlin/org/ostelco/at/jersey/Tests.kt]: Thanks, I will fix this

**Pull request merged**